app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -247,12 +248,16 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
     }
 
     private fun hideSystemUI() {
-        window.decorView.systemUiVisibility = (View.SYSTEM_UI_FLAG_LAYOUT_STABLE
-                or View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
-                or View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
-                or View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
-                or View.SYSTEM_UI_FLAG_FULLSCREEN
-                or View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY)
+        if (Build.VERSION.SDK_INT >= 28 && isCutout())
```

Comment:
Can we use the constant for the SDK version (`Build.VERSION_CODES.P`)?
Also I think it would be better if you move the SDK check inside the `isCutout()` check.

Reply:
I just deleted the SDK check because unnecessary